project-base/storefront/components/Layout/Header/MenuIconic/SalesRepresentative.tsx

```diff
@@ -0,0 +1,69 @@
+import { MailIcon } from 'components/Basic/Icon/MailIcon';
+import { SalesPhoneIcon } from 'components/Basic/Icon/SalesPhoneIcon';
+import { Image } from 'components/Basic/Image/Image';
+import { useCurrentCustomerData } from 'connectors/customer/CurrentCustomer';
+import { TypeImage } from 'graphql/types';
+import useTranslation from 'next-translate/useTranslation';
+
+export const SalesRepresentative: FC = () => {
+    const { t } = useTranslation();
+    const currentCustomerData = useCurrentCustomerData();
+    const salesRepresentative = currentCustomerData?.salesRepresentative;
+
+    if (!salesRepresentative) {
+        return null;
+    }
+
+    const { telephone, email } = salesRepresentative;
+
+    return (
+        <div className="flex items-start gap-4 w-full pt-4">
+            {salesRepresentative.images.length && (
+                <Image
+                    alt={t('Need advice?')}
+                    className="w-12 h-12 rounded-full"
+                    height={100}
+                    src={getRandomImage(salesRepresentative.images).url}
+                    width={100}
+                />
+            )}
+            <div className="font-medium dark:text-white">
+                <p className="text-lg font-semibold font-secondary leading-7">
+                    {getFullname(salesRepresentative.firstName, salesRepresentative.lastName)}
+                </p>
+                <p className="text-xs font-semibold font-secondary text-skyBlue uppercase tracking-wider">
+                    {t('Your sales representative')}
+                </p>
+                <div className="flex gap-2 items-center mt-1">
+                    <SalesPhoneIcon className="w-6 h-6" />
+                    <a className="no-underline text-sm text-black" href={`tel:${telephone}`}>
+                        {formatPhoneNumber(telephone)}
+                    </a>
+                </div>
+                <div className="flex gap-2 items-center mt-1">
+                    <MailIcon className="w-6 h-6" />
+                    <a className="no-underline text-sm text-black" href={`mailto:${email}`}>
+                        {email}
+                    </a>
+                </div>
+            </div>
+        </div>
+    );
+};
+
+const getFullname = (firstName?: string | null, lastName?: string | null): string => {
+    if (!firstName || !lastName) {
+        return firstName ?? lastName ?? '';
+    }
+    return `${firstName} ${lastName}`;
+};
+
+const formatPhoneNumber = (phoneNumber?: string | null, countryCallingCode?: number | null): string => {
+    return phoneNumber && phoneNumber.length === 9
+        ? phoneNumber.replace(/(\d{3})(\d{3})(\d{3})/, `${countryCallingCode ? '+' + countryCallingCode : ''} $1 $2 $3`)
+        : phoneNumber ?? '';
+};
+
+const getRandomImage = (images: TypeImage[]): TypeImage => {
+    return images[Math.floor(Math.random() * images.length)];
+}
```

Comment:
I would rather avoid this by having a single image.

Reply:
Yes we have agreed with @AndrejBlaho to change it to a single image to simplify.